src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -41,7 +44,11 @@ public class Shooter extends SubsystemBase {
 
     private static InterpolatingDoubleTreeMap metersToRPM = new InterpolatingDoubleTreeMap();
 
-    public Trigger shooterReadyTrigger = new Trigger(shooterPID::atSetpoint);
+    LEDs leds = LEDs.getInstance();
+
+    public Trigger shooterReadyTrigger = new Trigger(shooterPID::atSetpoint)
+            .onTrue(leds.applyPatternCommand(SOLID, GREEN.color))
+            .onFalse(new InstantCommand(()-> leds.restoreLEDs()));
```

Comment:
add a factory method that returns this, instead of creating here a new `InstantCommand`